nemo/collections/nlp/models/language_modeling/megatron_t5_prompt_learning_model.py

```diff
@@ -74,6 +75,15 @@ class MegatronT5PromptLearningModel(MegatronBasePromptLearningModel):
     def __init__(self, cfg: DictConfig, trainer: Trainer):
         super().__init__(cfg, trainer)
 
+        # define validation metric
```

Comment:
Can this logic be moved to the base class?

Reply:
moved to the upper class, but I get error for GPT model which I added it in the init_model function.